### PR TITLE
solution

### PR DESCRIPTION
**Pull request by pasha1932**
[DEMO LINK](https://pasha1932.github.io/react_list-of-goods-js/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -14,33 +16,96 @@ export const goodsFromServer = [
...
+          })}
+          onClick={() => setSortField(SORT_FIELD_LENGTH)}
+        >
+          Sort by length
+        </button>
+
+        <button
+          type="button"
+          className={cn('button', 'is-warning', {
+            'is-light': reversed !== true,
```
Comment (volodymyr-soltys97) [inline src/App.jsx:81]:
```suggestion
            'is-light': !reversed,
```

File: src/App.jsx [hunk 2]
```diff
@@ -14,33 +16,96 @@ export const goodsFromServer = [
...
+          })}
+          onClick={handleReverse}
+        >
+          Reverse
+        </button>
+
+        {sortField !== '' || reversed ? (
```
Comment (volodymyr-soltys97) [inline src/App.jsx:88]:
```suggestion
        {sortField || reversed ? (
```

File: src/App.jsx [hunk 3]
```diff
@@ -14,33 +16,96 @@ export const goodsFromServer = [
...
+          <button
+            type="button"
+            className="button is-danger is-light"
+            onClick={() => {
+              setSortField('');
+              setReversed(false);
+            }}
```
Comment (volodymyr-soltys97) [inline src/App.jsx:95]: Move this logic from JSX to the helper function and use it here

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's make your code better

**Review by lerastarynets — APPROVED** [review]
You're on the right track!

**Pull request closed without merging**